client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then
+	sudo -v 
+	sudo -K
+	python settingspanel.py
```

Comment:
I have `python3` as default Python installation, and your code is not compatible with Python3.

Please make the code compatible or call `python2`

> SyntaxError: Missing parentheses in call to 'print'